### PR TITLE
Added option to not connect to the BLE device upon discovery before the device is available

### PR DESCRIPTION
**Pull request by bryanreed**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+!default.pbxuser
+*.mode1v3
+!default.mode1v3
+*.mode2v3
+!default.mode2v3
+*.perspectivev3
+!default.perspectivev3
 xcuserdata
-Sample/Pods/**
-Sample/build/
```
Comment (MattCBowman) [inline .gitignore:3]: Can you verify that the sample project won't appear when you run pod install and when you build it? I don't want this to miss those directories

**Comment by MattCBowman** [comment]
Please also update the docs to mention the new boolean value you've added. Specifically, update the README.md file

File: .gitignore (hunk 1 above)
Comment (bryanreed) [inline .gitignore:3]: verified

**Comment by bryanreed** [comment]
Updated .md file

**Pull request merged**